src/libraries/System.Security.Cryptography.X509Certificates/src/Resources/Strings.resx

```diff
@@ -295,6 +295,12 @@
   <data name="Cryptography_X509_PKCS7_Unsupported" xml:space="preserve">
     <value>PKCS#7 certificate format is not supported on this platform.</value>
   </data>
+  <data name="Cryptography_X509_PKCS12_PersistKeySetNotSupported" xml:space="preserve">
+    <value>PKCS#12 persist key set flag is not supported on this iOS.</value>
+  </data>
+  <data name="Cryptography_X509_PKCS12_ExportableNotSupported" xml:space="preserve">
+    <value>PKCS#12 exportable flag is not supported on this iOS.</value>
```

Comment:
"this iOS" doesn't sounds clear to me, and since this probably affects iPadOS too, might as well just stick to "platform" as used in other messages.

```suggestion
    <value>PKCS#12 persist key set flag is not supported on this platform.</value>
  </data>
  <data name="Cryptography_X509_PKCS12_ExportableNotSupported" xml:space="preserve">
    <value>PKCS#12 exportable flag is not supported on this platform.</value>
```

Reply:
Oops, I didn't pay attention to grammar when changing `this platform` to `iOS`. Will change back to platform to be more general. Thanks for the catch :)